scylla-cdc/src/log_reader.rs

```diff
@@ -1,3 +1,20 @@
+//! A module containing the logic behind reading and consuming the data.
+//!
+//! Only one stream generation is being read at a time,
+//! while data from each stream is being read concurrently by different StreamReaders.
+//!
+//! The reading is done by reading all CDC rows added in a time window
+//! (not longer than `window_size`), then waiting for amount of time equal to `sleep_interval`
+//! before reading the next window.
+//! Before a newly added row is being read by the reader,
+//! a set amount of time, represented by `safety_interval`, must pass.
```

Comment:
Please rephrase this section so that it doesn't describe "the reading" but rather what a `StreamReader` is doing. Otherwise it might suggest that stream readers are synchronized in some way and they read a time window at the same time which isn't necessarily true.